<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_ContentView</class>
 <widget class="QWidget" name="page_ContentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1860</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget_Content_Left" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="planetLabel">
             <property name="text">
              <string>Planet</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="planetComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="renderResolutionLabel">
             <property name="text">
              <string>Render Resolution</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="renderResolutionComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="coordinateSystemLabel">
             <property name="text">
              <string>Coordinate System</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="coordinateSystemComboBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="sampleModeLabel">
             <property name="text">
              <string>Sample Mode</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="sampleModeComboBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="outputResolutionLabel">
             <property name="text">
              <string>Output Resolution</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="outputResolutionComboBox"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="displayModeLabel">
             <property name="text">
              <string>Display Mode</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="displayModeComboBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="displayLayerLabel">
             <property name="text">
              <string>Display Layer</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="displayLayerComboBox"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="enableGridLabel">
             <property name="text">
              <string>Enable Grid</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="enableGridCheckBox">
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="enableCrosshairLabel">
             <property name="text">
              <string>Enable Crosshair</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="enableCrosshairCheckBox">
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QListView" name="listWaypoints"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Waypoints</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="enableWaypointsLabel">
             <property name="text">
              <string>Enable Waypoints</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="enableWaypointsCheckBox">
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>250</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lbl_planetDetails">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_currentStatus">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renderButton">
           <property name="text">
            <string>Render</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_Content_Viewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPlanetViewer" name="renderOutput"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPlanetViewer</class>
   <extends>QGraphicsView</extends>
   <header>starfab.gui.widgets.planets.planet_viewer</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
